mautic/mautic
# Revert media/js/mautic-form-src.js

This reverts commit 6856ab86e9de7ca0f761b5dfc4b335468f3931d7.



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Seems like we accidentaly remove media/js/mautic-form-src.js https://github.com/mautic/mautic/commit/6856ab86e9de7ca0f761b5dfc4b335468f3931d7
But we still need it for dev enviroment, preview form.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  On dev enviroment, go to preview form
3. Form is not submittend properly and brower console return error 404
